OLCNE/scripts/provision.sh

```diff
@@ -344,32 +422,48 @@ passwordless_ssh() {
   msg "Allow passwordless ssh between VMs"
   # Generate common key
   if [[ ! -f /vagrant/id_rsa ]]; then
-    msg "Generating shared SSH keypair"
-    echo_do ssh-keygen -t rsa -f /vagrant/id_rsa -q -N "''" -C 'vagrant@olcne'
+    msg "Generating shared SSH keypair in PEM format"
+    echo_do ssh-keygen -m PEM -t rsa -f /vagrant/id_rsa -q -N "''" -C "'vagrant@olcne'"
   fi
+  # Generate known_hosts
+  if [[ ! -f /vagrant/known_hosts ]]; then
+    msg "Generating shared SSH Known Hosts file"
+    echo_do touch /vagrant/known_hosts
+  fi  
   # Install private key
   #echo_do mkdir -p /root/.ssh
   echo_do cp /vagrant/id_rsa ~/.ssh
   echo_do cp /vagrant/id_rsa.pub ~/.ssh
   # Authorise passwordless ssh
   #echo_do cp /vagrant/id_rsa.pub /root/.ssh/authorized_keys
-  echo_do sh -c "'cat /vagrant/id_rsa.pub >> ~/.ssh/authorized_keys'"
+  #echo_do sh -c "'cat /vagrant/id_rsa.pub >> ~/.ssh/authorized_keys'"
+  echo_do "cat /vagrant/id_rsa.pub >> ~/.ssh/authorized_keys"
   # Don't do host checking
-  cat > ~/.ssh/config <<-EOF
-	Host operator* master* worker* 192.168.99.*
-	  StrictHostKeyChecking no
-	  UserKnownHostsFile /dev/null
-	  LogLevel FATAL
-	EOF
+  # cat > ~/.ssh/config <<-EOF
```

Comment:
Shouldn't this be removed instead of just commenting it?

Reply:
Thank you @rafabene. Removed now.